README.md

```diff
@@ -227,6 +227,27 @@ list.addEventListener('rangechange', (event) => {
 });
 ```
 
+### Scrolling
+
+`<virtual-list>` needs to be sized in order to determine how many items should be rendered. Its default size is:
+```css
+virtual-list {
+  /* https://www.w3.org/TR/CSS2/visudet.html#inline-replaced-width */
+  width: 300px;
+  /* https://www.w3.org/TR/CSS2/visudet.html#inline-replaced-height */
+  height: 150px;
+}
+```
+
+Main document scrolling will be achievable through [`document.rootScroller`](https://www.chromestatus.com/feature/4533350404456448)
+```html
+<virtual-list style="height: 100vh"></virtual-list>
+<script type="module">
+  import './node_modules/virtual-list/virtual-list-element.js';

```

Comment:
Most of our other examples don't include this import; the one that does uses `std:virtual-list|https://some.cdn.com/virtual-list.js`. I think we should leave it out.